app/code/Magento/Customer/Model/Vat.php

```diff
@@ -212,6 +212,11 @@ public function checkVatNumber($countryCode, $vatNumber, $requesterCountryCode =
                 $gatewayResponse->setRequestMessage(__('Please enter a valid VAT number.'));
             }
         } catch (\Exception $exception) {
+            $this->logger->error(
+                __('VAT Number validation failed with message: %1', $exception->getMessage()),
```

Comment:
Logged data does not need to be translated

Reply:
Makes sense to me (I got incorrectly inspired by some of the other translated logged data in Magento's codebase).

Anyway, fixed now!